### DISCUSSION TITLE
Feature Suggestion: Allow Terminal Input Pane to be Pinned to Top and Bottom in Hyper.js

### DISCUSSION THREAD

**Discussion by erickvasm** [vercel/hyper#7966] [Ideas]
Hey Hyper.js Team,

I wanted to share an idea that I think could make Hyper.js even cooler: what if we allow the **terminal input pane to be pinned to the top or bottom of the window**?

Reasons for Implementing the Improvement:

1.  Eases Command Writing: No more scrolling to access the input pane. Just write your commands smoothly.
2.  Improves Visibility: By keeping the input pane fixed, you'll always have a clear view of the results of your commands.
3.   Reduces Visual Fatigue: Less scrolling means less visual fatigue and greater comfort.
4. Increases Productivity: By minimizing distractions related to scrolling, users can work more efficiently.

Implementation Sketch:

In terms of implementation, we could add an option in the Hyper.js settings that allows the user to pin the input pane to the top or bottom of the window. 

![img](https://github.com/vercel/hyper/assets/55751600/09f850be-aa90-4a8f-a9ab-d550c00a53a3)

[References](https://github.com/warpdotdev/warp/issues/22)

What do you think of the idea? I believe it would be a very useful feature for all Hyper.js users!

**Comment by louis-l** [comment]
+1
